sections/sprint-4/conclusion.tex

```diff
@@ -0,0 +1,66 @@
+\section{Sprint 4 conclusion}\label{sec:sprint4conclusion}
+This section concludes sprint 4 and will summarize the progress that has been made on the game and discuss the retrospective meeting conducted at the end of sprint 4.
+
+\subsection{Current product}
+The two current states of the game can be seen in \autoref{fig:sprint-4-state-of-game}.
+We are currently able to connect to the host in the lobby, as seen on \autoref{fig:sprint-4-lobby}, which will lead to \autoref{fig:sprint-4-game} when the host has started the game.
+\begin{figure}[H]
+    \centering
+    \begin{subfigure}{.5\textwidth}
+        \centering
+        \includegraphics[width=1\linewidth]{figures/sprint-4-lobby.PNG}
+        \caption{The lobby where you need to input the IP address to connect.}
+        \label{fig:sprint-4-lobby}
+    \end{subfigure}
+    \begin{subfigure}{.4\textwidth}
+        \centering
+        \includegraphics[width=.8\linewidth]{figures/sprint-4-game.PNG}
+        \caption{The current playing field with goal zones, goal score and players.}
+        \label{fig:sprint-4-game}
+    \end{subfigure}
+    \caption{The current state of the game}
+    \label{fig:sprint-4-state-of-game}
+\end{figure}
+
+\subsubsection{Pozyx}
+There are not many Pozyx programming tasks left, but there are some bugs that needs to be fixed during the next sprint.
+\\
+During the initial test of the program described in \autoref{sec:initial-test} several exceptions occurred, which were caused by the input data not being correctly formatted.
+This should be fixed in the following sprint.
+Problems with the firmware were also discovered in the initial test, where Pozyx would not send positional data to the host, but only coordinates of $0.0$.
+
+\subsubsection{Game}
+If the problems with Pozyx gets fixed, the game should have implemented the majority of the requirements defined for the MVP in \autoref{sec:MVP}.
+However, in the current iteration of the game, the teams are unable to win the game.
+This is a highly prioritized Unity task, which we intend to implement in the last sprint.
+
+\subsubsection{Networking}
+To be able to win the game, it is necessary for the host to be able to send a message that the game has ended to the clients.
+Therefore a package needs to be specified for this purpose to be implemented along with the ability to win a game.
```

Comment:
skriv måske noget med at det også indebærer at hosten definerer en mængde mål for at vinde og sp check på det når der mål i guess, så er der lidt mere 🤔